src/EventStore.ClusterNode/ClusterVNodeHostedService.cs

```diff
@@ -77,12 +77,12 @@ internal class ClusterVNodeHostedService : EventStoreHostedService<ClusterNodeOp
 		protected override void PreInit(ClusterNodeOptions options) {
 			base.PreInit(options);
 
-			if (options.Db.StartsWith("~") && !options.Force) {
+			if (options.Db.StartsWith("~")) {
```

Comment:
the error message is now wrong, but i would rather just ditch support for this entirely.  What is the reason for it being supported in the first place?